### PR TITLE
SceneView: add SceneViewInteractionOptions

### PR DESCRIPTION
**Pull request by changanxian**
Adds ScenViewInteractionOptions Params to the composable SceneView.

ad hoc testing on a SceneView:

```
 SceneView(
                modifier = Modifier.fillMaxSize(),
                arcGISScene = scene,
                sceneViewInteractionOptions = SceneViewInteractionOptions(isRotateEnabled = false)
            )
```

### CODE REVIEW

File: toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/SceneView.kt [hunk 1]
```diff
@@ -51,6 +52,7 @@ import kotlinx.coroutines.launch
  * @param viewpointOperation a [SceneViewpointOperation] that changes this SceneView to a new viewpoint
  * @param graphicsOverlays the [GraphicsOverlayCollection] used by this composable SceneView
  * @param sceneViewProxy the [SceneViewProxy] to associate with the composable SceneView
+ * @param sceneViewInteractionOptions a [SceneViewInteractionOptions] that configures the interaction options for the composable SceneView
```
Comment (gunt0001) [inline toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/SceneView.kt:55]: to be consistent with the corresponding doc on composable MapView:
```suggestion
 * @param sceneViewInteractionOptions the [SceneViewInteractionOptions] used by this composable SceneView
```

**Review by gunt0001 — APPROVED** [review]
@changanxian - looks good, one comment re doc

**Review by rchintapalli — APPROVED** [review]
LGTM

**Comment by changanxian** [comment]
closed this one due to an accident merging from main->this

**Pull request closed without merging**